### DISCUSSION TITLE
Github Pages branch not displaying updated

### DISCUSSION THREAD

**Discussion by JManyama** [community/community#23175] [Other Feature Feedback, Questions, & Ideas]
<p>Hi All,</p>
<p>I’ve been deploying a Jekyll site via CircleCI to a gh-pages branch, and previously gh-pages would display the new changes I committed to my repo. Now however, the gh-pages branch does not get updated. The only thing that does get updated are the files I commit.</p>
<p>For example, below is a screenshot of the index.html file that was updated in my Main branch. It shows as updated 1 hour ago.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/6/c6b054e9420c6c88227bc161d8ac057737037c50.png" data-download-href="/uploads/short-url/slGkkZ7SglUxD6yd3kyGFVD2Itq.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/74265784/181075442-8b652055-d608-48f8-aad3-80783b71c5e9.png" alt="image" data-base62-sha1="slGkkZ7SglUxD6yd3kyGFVD2Itq" width="690" height="177" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/6/c6b054e9420c6c88227bc161d8ac057737037c50_2_690x177.png, https://aws1.discourse-cdn.com/github/optimized/3X/c/6/c6b054e9420c6c88227bc161d8ac057737037c50_2_1035x265.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/c/6/c6b054e9420c6c88227bc161d8ac057737037c50.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/6/c6b054e9420c6c88227bc161d8ac057737037c50_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1182×304 7.15 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>This one is of my gh-pages branch, shows as updated 2 hours ago.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/7/d723999ab7bba2e8abf226988a3f381fc57fc530.png" data-download-href="/uploads/short-url/uHcVAqvG8vgaxdcOimnfVzuakwM.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/74265784/181075443-41daab7b-9f84-428a-9896-e69246527bf8.png" alt="image" data-base62-sha1="uHcVAqvG8vgaxdcOimnfVzuakwM" width="690" height="157" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/7/d723999ab7bba2e8abf226988a3f381fc57fc530_2_690x157.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/7/d723999ab7bba2e8abf226988a3f381fc57fc530_2_1035x235.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/7/d723999ab7bba2e8abf226988a3f381fc57fc530.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/7/d723999ab7bba2e8abf226988a3f381fc57fc530_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1162×266 11.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I’m not sure why the gh-pages branch updates with each new commit, and why it doesn’t at other times, but its causing my rendered site not to display the most recent changes.</p>
<p>Any idea what I could be missing here?</p>
<p>Thanks.</p>

**Comment by kingthorin** [comment]
<p>Hard to say without being able to look at the repo. You’re probably pushing to the wrong branch just as a guess.</p>

**Comment by otter-computer** [comment]
<p>Can you post a link to the repository so we can take look?</p>

**Comment by JManyama** [comment, marked as answer]
<p>The issue is resolved. It appears to have been caused by 2 Github account email addresses being used on the same repo. On my CircleCI config.yml, my Git email address was different than what was being used in my Git CLI.</p>
<p>Once I made the Github Account email address, the Git email and the config.yml email address all match, the GH pages branch has been working correctly.</p>
